docs/advanced/customization.md

```diff
@@ -65,7 +65,7 @@ View the Container Linux Config [format](https://coreos.com/os/docs/1576.4.0/con
 
 Write Container Linux Configs *snippets* as files in the repository where you keep Terraform configs for clusters (perhaps in a `clc` or `snippets` subdirectory). You may organize snippets in multiple files as desired, provided they are each valid.
 
-[AWS](../flatcar-linux/aws.md#cluster) and [Azure](../flatcar-linux/azure.md#cluster) clusters allow populating a list of `controller_clc_snippets` or `worker_clc_snippets`.
+[AWS](../flatcar-linux/aws.md#cluster), [Azure](../flatcar-linux/azure.md#cluster) and [Packet](../flatcar-linux/packet.md#cluster) clusters allow populating a list of `controller_clc_snippets` or `worker_clc_snippets`.
```

Comment:
Oh, using `worker_clc_snippets` in the docs here seems old. For providers that we support worker pools (like AWS or Packet), `clc_snippets` is used in the terraform worker module.

The example is out of date, too. The example doesn't use worker pool (and it mentions `clc_snippets` for baremetal only). We can keep it like that in this PR, and I can create another issue to update the docs with worker pool info (so we keep the scope of this PR focused). IOW, let's add Packet to that list as this PR does, but update the example as part of another issue.

Will that work for you?